include/boost/fixed_string/fixed_string.hpp

```diff
@@ -37,20 +37,17 @@ template<
     std::size_t N,
     typename CharT = char,
     typename Traits = std::char_traits<CharT>>
-class fixed_string
+class fixed_string : public detail::fixed_string_base<N, CharT>

```

Comment:
This needs to be a private derivation, and you need to protect it from the docs using `#ifndef GENERATING_DOCUMENTATI)ON` or whatever